virtwho/virt/ahv/ahv_interface.py

```diff
@@ -514,14 +514,17 @@ def get_vms_uuid(self, version):
         self._logger.info("Getting the list of available vms")
         is_pc = True if version == 'v3' else False
         vm_uuid_list = []
-        length = 0
+        length = 100
```

Comment:
Could you please add constants 100 as a class attribute (something like `NUM_OF_REQUESTED_VMS`) ?